lib/JsonLdDocumentLoader.js

```diff
@@ -18,8 +19,51 @@ class JsonLdDocumentLoader {
     this.documents.set(url, document);
   }
 
+  /**
+   * Adds a custom protocol handler to the loader.
+   *
+   * @example
+   * // Using did-io's CachedResolver instance as a did: protocol handler
+   * jld.addProtocolHandler({protocol: 'did', handler: cachedResolver});
+   *
+   * @param {string} protocol - Protocol id, such as 'did', 'https', etc.
+   * @param {{get: Function}} handler - Protocol handler object, exposes an
+   *   async `get({url})` method.
+   */
+  addProtocolHandler({protocol, handler} = {}) {
+    this.protocolHandlers.set(protocol, handler);
+  }
+
+  /**
+   * Adds a DID resolver (either a method-specific resolver, or a general driver
+   * harness) to protocol handlers.
```

Comment:
I think we need to be more clear that you can't just add multiple drivers -- it won't work. If you want multiple drivers, you need to use the "general driver harness" you mention:

```suggestion
   * Adds a DID resolver (either a method-specific resolver to support just a
   * single DID method, or a general driver harness to support multiple
   * DID methods) to protocol handlers.
```

Reply:
Changed, thanks.